src/sentry/release_health/base.py

```diff
@@ -1,9 +1,24 @@
+from __future__ import annotations
+
 from datetime import datetime
-from typing import Literal, Mapping, Optional, Sequence, Set, Tuple, TypedDict, TypeVar, Union
+from typing import (
+    TYPE_CHECKING,
+    Literal,
+    Mapping,
+    Optional,
+    Sequence,
+    Set,
+    Tuple,
+    TypedDict,
+    TypeVar,
+    Union,
+)
 
-from sentry.snuba.sessions_v2 import QueryDefinition
 from sentry.utils.services import Service
 
+if TYPE_CHECKING:
+    from sentry.snuba.sessions_v2 import QueryDefinition
```

Comment:
This caused some problems when we started loading `release_health` as an app, but luckily we're only using this for type checking.